### ISSUE TITLE
Questions about the loss curve in pretrain

### ISSUE THREAD

**Issue by jyC23333** [OpenGVLab/InternVL#276]
Hi, Could you please share the pretrain and SFT loss curve of internVL series?

I can not get a make-sense loss curve when pretraining the model, the loss seems to be stuck at 3. Following the instruction [here](https://arxiv.org/abs/2404.16821), adapter and ViT are unfreezed while training.

By the way, I have a mount of training data in pretrain stage.

![image](https://github.com/OpenGVLab/InternVL/assets/110331827/cfd04c08-010e-412b-b992-878f898d132b)

**Comment by czczup** [comment]
Hi, you can see the SFT training curve of InternVL-Chat-V1-5 here: https://huggingface.co/OpenGVLab/InternVL-Chat-V1-5/tensorboard. Regarding the pretraining curve, I can briefly explain that my loss is around 1.8.

Could you please provide more information about your experiment, such as which language model you used, the batch size, the data, etc.? These factors can significantly affect the loss.

**Comment by jyC23333** [comment]
@czczup Hi! I try some hyper params these days, this is my newest pretraining  results:
![image](https://github.com/user-attachments/assets/4f3c27c4-1587-442c-9a21-0a828475aea8)
As above, the loss seems to be stuck at around 2.5.

Currently, my training setting is blow: 
```
  --max_dynamic_patch 12 \
  --drop_path_rate 0.1 \
  --bf16 True \
  --num_train_epochs 1 \
  --per_device_train_batch_size 2 \
  --gradient_accumulation_steps 64 \
  --learning_rate 1e-4 \
  --weight_decay 0.05 \
  --warmup_ratio 0.03 \
  --lr_scheduler_type cosine \
  --dynamic_image_size True \
  --use_thumbnail True \
```
I used 64 gpus for training, which means that the global batchsize reaches ```2*64*64 = 8192```

I used about 1M Chinese English image-text pairs which are from open-sourced data such as laion、COCO.


Could you please give some suggestions? Thanks so much!

**Comment by czczup** [comment]
The pretrain loss of language models of different sizes can vary significantly. For example, the loss of a 2B model is much higher than that of a 26B model.

**Comment by czczup** [comment]
Yes, this phenomenon is observed during the pre-training phase, and loss decreases very slowly in the later stages of pre-training.

**Comment by jyC23333** [comment]
Thanks!
